pkgs/applications/science/electronics/nxpmicro-mfgtools/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  bzip2,
+  libusb1,
+  libzip,
+  openssl,
+  pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  name    = "nxpmicro-mfgtools-1.3.191";
+  version = "uuu_1.3.191";
```

Comment:
```suggestion
  pname = "nxpmicro-mfgtools";
  version = "1.3.191";
```

Reply:
According to https://nixos.org/nixpkgs/manual/#sec-using-stdenv , pname with set the name with the right version.